src/components/BreedsList.vue

```diff
@@ -182,31 +183,33 @@ export default {
     },
 
     toggleCard(breed, id) {
-      this.indexClicked = id;
+      this.indexClicked++;
       breed.isActive = !breed.isActive;
       if (breed.isActive) {
         this.lastBreedState.push(breed);
```

Comment:
ti pushaš cel `breed` object, a bi lahko samo ID zapisal mogoče?

Reply:
tukaj sem pushov cel `breed `object, ker potem lastBreedState uporabim v drugi metodi `scrollToLastOpenCard`, ki scrola na sredino zadnji odprti breed. in tukaj potem rabim breed.name. Razen, če bi se nekako drugače lotil znotraj te metode scrollToLastOpenCard? Kaj misliš?
```
 scrollToLastOpenCard() {
      if (this.$route.name === "breeds" && this.lastBreedState.length > 0) {
        this.displayedBreeds.filter((breed, id) => {
          if (
            this.lastBreedState[this.lastBreedState.length - 1].name ===
            breed.name
          ) {
            this.$refs.resultListContainer.children[id].scrollIntoView({
              behavior: "smooth",
              block: "center",
            });
          }
        });
      }
    },
```

Reply:
V tvojem primeru je vredu ker imas majhne objekte in jih ni veliko.. v primeru ce bi imel vec tisoc objektov bi shranjeval samo IDje in pol kjer bi kaj rabu ponovno najdu referenco in property.. 

Lahko pa tudi v tvojem primeru passas v last breed samo kar rabis.. sepravi ID in name, ce prav razumem